official/utils/logging/perf_hooks_helper.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Hooks helper to return a list of TensorFlow hooks for training by name."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import perf_hooks
+import tensorflow as tf
+
+
+def get_train_hooks(names, **kwargs):
+  """Factory for getting a list of TensorFlow hooks for training by name.
+
+  Args:
+    names: case-insensitive, comma-separated string with names of desired hook
+      classes. Allowed: LoggingTensorHook, ProfilerHook, ExamplesPerSecondHook
+    **kwargs: a dictionary of arguments to the hooks
```

Comment:
nit: I think it is standard to just list `kwargs` in the Args section, rather than `**kwargs`.